frontend/src/components/utils/Footer.jsx

```diff
@@ -1,154 +1,176 @@
-            <span className="need-help-contact">Questions?</span>
-            <span className="need-help-contact need-help-contact-bottom">
-              Join the Community
-            </span>
+          <Col
+            className="align-self-center footer-link text-md-left text-center pl-0"
+            xs="12"
+            md="6"
+          >
+            © {new Date().getFullYear()} NEAR Inc. All Rights Reserved.
+            <br />
+            <a
+              className="footer-link-href"
+              href="https://near.org/privacy/"
+              onClick={() =>
+                Mixpanel.track("Footer click to see terms of service")
+              }
+            >
+              Terms of Service
+            </a>
+            &nbsp;|&nbsp;
+            <a
+              className="footer-link-href"
+              href="https://near.org/privacy/"
+              onClick={() =>
+                Mixpanel.track("Footer click to see privacy policy")
+              }
+            >
+              Privacy Policy
+            </a>
           </Col>
         </Row>
-      </a>
-    </div>
-    <style jsx global>{`
-      .footer-container {
-        background-color: #f8f8f8;
-        font-size: 12px;
-        line-height: 40px;
-        color: #999999;
-        height: 118px;
-        display: flex;
-        justify-content: space-between;
-        flex-direction: row;
-      }
-
-      .footer-container > .col-12 {
-        min-height: 70px;
-        padding-top: 20px;
-        padding-bottom: 25px;
-      }
-
-      .nearlogo-wrapper {
-        padding: 30px;
-        width: 70%;
-        font-weight: normal;
-      }
-
-      .footer-link-href {
-        color: #999999;
-        text-decoration: underline;
-      }
-
-      .footer-link {
-        line-height: 20px;
-      }
-
-      .footer-container .near-logo {
-        height: 60px;
-        fill: #acacac;
-      }
-
-      .footer-help {
-        text-align: left;
-        line-height: 24px;
-        padding-top: 22px;
-        width: 175px;
-      }
-
-      .help-image {
-        position: relative;
-        z-index: 2;
-        left: 10px;
-        top: -7px;
-      }
-
-      .need-help-contact {
-        font-size: 18px;
-        font-weight: 900;
-        color: #999999;
-        text-decoration: none !important;
-      }
-
-      .footer-help-link:hover {
-        text-decoration: none !important;
-      }
-
-      .need-help-contact-bottom {
-        color: #0072ce !important;
-        display: block;
-        font-weight: 600;
-      }
-
-      .near-logo {
-        width: 100px;
-      }
-
-      .nearkat-wrapper {
-        background: #ffffff;
-        border: 1px solid #dadada;
-        box-sizing: border-box;
-        border-radius: 3px;
-        width: 368px;
-        height: 92px;
-      }
-
-      @media (max-width: 780px) {
+      </div>
+      <div className="nearkat-wrapper">
+        <a
+          className="footer-help-link"
+          href="http://near.chat"
```

Comment:
We lost `target="_blank"` here. Please, add `target="_blank"` and `rel="noopener"` ([read more](https://mathiasbynens.github.io/rel-noopener/)) to the external links